src/modules/header/dropdown-languages/dropdown-languages.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { Dropdown, DropdownProps, MenuProps } from "antd";
+import { CaretDownOutlined, CaretUpOutlined } from "@ant-design/icons";
+import classNames from "classnames";
+import styles from "../header.module.css";
+
+type LanguageType = "EN" | "DE" | "RU";
+
+const DropdownLanguages = () => {
+    const [isLanguagesOpen, setLanguagesOpen] = useState(false);
+    const [language, setLanguage] = useState<LanguageType>("EN");
+
+    const languages: MenuProps["items"] = [
+        {
+            label: <span style={{ padding: "12px", fontSize: "18px" }}>English</span>,
+            key: "EN",
+        },
+        {
+            label: <span style={{ padding: "12px", fontSize: "18px" }}>Deutsch</span>,
```

Comment:
as i know we are not going translate to german

Reply:
ok